doc/vpc_peering.md

```diff
@@ -0,0 +1,20 @@
+# VPC Peering
+
+## What
+
+We use VPC peering to allow tenants to access resources they manage in another VPC somewhere on AWS. We grant access to the VPC on a per-space basis, although there is a shortcut to grant access to all existing spaces in an org.
+
+## How
+
+1. Choose a subnet CIDR that we'll ask the tenant to use. Look in `paas-cf/terraform/prod.vpc_peering.json` for a list of current peerings, then choose the next highest subnet - e.g. if the highest subnet is `172.16.4.0/24`, ask the tenant to use `172.16.5.0/24`.
+2. Tell the tenant to create a new VPC with the subnet you have given them, and ask them to give you both the VPC ID, and the AWS account ID the VPC resides in.
+3. Create a new entry in `paas-cf/terraform/prod.vpc_peering.json` (or whichever environment they need to peer with). Deploy this to production.
+4. A CF security group will have been automatically created that gives access to the peered VPC, the name of the group will be `vpc_peer_$PEER_NAME`. `cf bind-security-group` this to the spaces that the tenant needs access from.
```

Comment:
Do we need to add something around informing the tenant that they'll need to raise support requests to enable this for any future spaces that need access.